tests/unittest/IO/volumeloader.cpp

```diff
@@ -93,7 +95,9 @@ TEST_CASE( "IO/VolumesLoader", "[IO]" ) {
         auto binsize = volumeData->binSize();
         REQUIRE( ( Math::areApproxEqual( binsize[0], 1._ra ) &&
                    Math::areApproxEqual( binsize[1], 1._ra ) &&
-                   Math::areApproxEqual( binsize[2], 1.4_ra ) ) );
+                   // if Scalar = double, since reader use float, increase epsilon by float/double
+                   // epsilon ratio
+                   Math::areApproxEqual( binsize[2], 1.4_ra, 436870912_ra ) ) );
```

Comment:
Couldn't we have something like this (with the proper syntax):
```
Math::areApproxEqual( binsize[2], 1.4_ra, epsilon<float>()/epsilon<double>() ) ) );
```

Reply:
yes we can